docs/root/start/quick-start/configuration-dynamic-filesystem.rst

```diff
@@ -8,6 +8,11 @@ protocol.
 
 When the files are changed on the filesystem, Envoy will automatically update its configuration.
 
+.. note::
+
+   Envoy only updates when the configuration file is replaced, and not when the file is edited in place.
+   It is implemented this way to ensure configuration consistency, more details refer to :ref:`Atomic swap for configuration file updates<config_runtime_symbolic_link_swap>`.
```

Comment:
The grammar is not quite right here, and i think it needs some spacing

```suggestion
   Envoy only updates when the configuration file is replaced, and not when the file is edited in place.
   
   It is implemented this way to ensure configuration consistency.
   
   Please see the section on :ref:`atomic swaps <config_runtime_symbolic_link_swap>` for further information on how runtime configuration is updated.
```

Reply:
Done.